### ISSUE TITLE
Create User Stats Page

### ISSUE THREAD

**Issue by fzyang1227** [Code-4-Community/speak-for-the-trees-frontend#245]
Blocked by #239 

**Why**
- Allow users to see their own contributions to the website

**What to do**
- Build out page according to #239 
- Unique page according to each user accessible to the user and ADMIN+ users
    - redirect NONE to 'login' and other STANDARD to 'not found'
- Page includes:
    - The user’s name and contact information
    - How many and which trees the user has currently adopted
    - For each tree, how many and the kinds of stewardship activities the user has performed in the past specified timeframe

**Comment by huang0h** [comment]
Closing as the page is no longer needed
